### PR TITLE
build file cleanup and docker credentials

### PR DESCRIPTION
**Pull request by rcordeau**
This change is [Reviewable](https://reviewable.io/reviews/smarterapp/rdw_ingest/230)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,3 +1,17 @@
+/*
```
Comment (malaffoon) [inline build.gradle]: @jtreuting did we ever finalize the requirement of putting the copyright in every file or only in the LICENSE file in the project?

Comment (rcordeau) [inline build.gradle]: IAT is using copyright at least in the build.gradle.

Comment (malaffoon) [inline build.gradle]: And TDS has them in every source file. 

Doesn't mean that's the necessary and/or correct thing to do.

Comment (rcordeau) [inline build.gradle]: Removed copyright statement.

**Pull request merged**